Mathlib/Tactic/SplitIfs.lean

```diff
@@ -0,0 +1,198 @@
+/-
+Copyright (c) 2018 Gabriel Ebner. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Gabriel Ebner, David Renshaw
+-/
+import Lean
+import Mathlib.Init.Logic
+import Mathlib.Tactic.SplitIfsAttr
+
+/-!
+Tactic to split if-then-else expressions.
+-/
+
+namespace Mathlib.Tactic
+
+open Lean Elab.Tactic Parser.Tactic Lean.Meta
+
+attribute [split_ifs_reduction] if_pos if_neg dif_pos dif_neg if_congr
+
+/-- Simulates the `all_goals` tactic combinator.
+-/
+private def evalAllGoals : TacticM Unit → TacticM Unit := fun tac => do
```

Comment:
This is already in `Mathlib/Tactic/Core.lean`, at the end.

Reply:
Ah, very good. Updated.